src/sidebar/services/analytics.js

```diff
@@ -2,38 +2,108 @@
 
 const VIA_REFERRER = /^https:\/\/(qa-)?via.hypothes.is\//;
 
-const globalGAOptions = function(win, settings) {
-  settings = settings || {};
-
-  const globalOpts = {
-    category: '',
-  };
+const events = {
```

Comment:
The `events` mapping was hoisted up from inside a function to be a module-level constant.